tasks/linters/yamllint/README.md

```diff
@@ -0,0 +1,67 @@
+# YAML Linting Task
```

Comment:
Same with this one, should we move it to `tasks/linters/yamllint/0.1/` directory?